docs/src/app/components/pages/components/AutoComplete/ExampleFilters.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import AutoComplete from 'material-ui/lib/auto-complete';
+
+const fruit = [
+  'Apple',
+  'Apricot',
+  'Avocado',
+  'Banana',
+  'Bilberry',
+  'Blackberry',
+  'Blackcurrant',
+  'Blueberry',
+  'Boysenberry',
+  'Cantaloupe',
+  'Currant',
+  'Cherry',
+  'Cherimoya',
+  'Cloudberry',
+  'Coconut',
+  'Cranberry',
+  'Damson',
+  'Date',
+  'Dragonfruit',
+  'Durian',
+  'Elderberry',
+  'Feijoa',
+  'Fig',
+  'Goji berry',
+  'Gooseberry',
+  'Grape',
+  'Raisin',
+  'Grapefruit',
+  'Guava',
+  'Huckleberry',
+  'Jabouticaba',
+  'Jackfruit',
+  'Jambul',
+  'Jujube',
+  'Juniper berry',
+  'Kiwi fruit',
+  'Kumquat',
+  'Lemon',
+  'Lime',
+  'Loquat',
+  'Lychee',
+  'Mango',
+  'Marion berry',
+  'Melon',
+  'Cantaloupe',
+  'Honeydew',
+  'Watermelon',
+  'Miracle fruit',
+  'Mulberry',
+  'Nectarine',
+  'Olive',
+  'Orange',
+  'Blood Orange',
+  'Clementine',
+  'Mandarine',
+  'Tangerine',
+  'Papaya',
+  'Passionfruit',
+  'Peach',
+  'Pear',
+  'Persimmon',
+  'Physalis',
+  'Plum',
+  'Pineapple',
+  'Pumpkin',
+  'Pomegranate',
+  'Pomelo',
+  'Purple Mangosteen',
+  'Quince',
+  'Raspberry',
+  'Salmonberry',
+  'Rambutan',
+  'Redcurrant',
+  'Salal berry',
+  'Satsuma',
+  'Star fruit',
+  'Strawberry',
+  'Squash',
+  'Tamarillo',
+  'Tamarind',
+  'Tomato',
+  'Ugli fruit',
+];
```

Comment:
I didn't mean remove the example. I just meant that it takes a long way to get to 14 lines of code that are component codes down there. If we can have like 6-10 fruits per line it will be easier to get down to the code needed for the component.

in other words instead of:
```js
const fruit = [
  'Apple',
  'Apricot',
  'Avocado',
  'Banana',
  'Bilberry',
  'Blackberry',
...
```

we should have this:
```js
const fruit = [
  'Apple',  'Apricot',  'Avocado',  'Banana',  'Bilberry',  'Blackberry',  'Blackcurrant',
  'Boysenberry',  'Cantaloupe',  'Currant',  'Cherry',  'Cherimoya',  'Cloudberry',
  'Coconut',  'Cranberry', ...
```

That's all :grin: